dist/platforms/ubuntu/steps/runsteps.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+#
+# Run steps
+#
+source /steps/set_extra_git_configs.sh
+source /steps/set_gitcredential.sh
+source /steps/activate.sh
+source /steps/build.sh
+source /steps/return_license.sh
```

Comment:
Thinking out loud: Using source here might not respect the shebangs in the scripts, but so far, everything uses bash so I think this is fine.

Reply:
Agreed, let's keep it this way as things are working fine. Extracting scripts like this is at least way more easier to read. Nicely done 👍